docs/release/updating.md

```diff
@@ -23,6 +22,9 @@ The recommended procedure for the upgrade is:
 
 - On all servers, admin nodes, and clients,
   perform the RPM update to the new DAOS fix level.
+  Set up the new DAOS repository, then run `yum update` (CentOS 7 or EL8),
+  `dnf update` (EL8), or `zypper update` (SLES/OpenSUSE)
+  to update the DAOS RPMs and their dependencies.
```

Comment:
(same comment as on the release/2.0 PR)
Is it worth adding a "!!! warning" note that if the yum/dnf "install" sub-command is used inadvertently (instead of update) that problems could occur?

Reply:
that was decided against (and that seems reasonable to me), response to the question in the other PR https://github.com/daos-stack/daos/pull/10104#discussion_r955867652